bom/platform-next/pom.xml

```diff
@@ -0,0 +1,1206 @@
+            </dependency>
+            <dependency>
+                <groupId>software.amazon.awssdk</groupId>
+                <artifactId>core</artifactId>
+                <version>${version.bom.software.amazon.awssdk}</version>
+            </dependency>
+            <dependency>
+                <groupId>software.amazon.awssdk</groupId>
+                <artifactId>crt-core</artifactId>
+                <version>${version.bom.software.amazon.awssdk}</version>
+            </dependency>
+            <dependency>
+                <groupId>software.amazon.awssdk</groupId>
+                <artifactId>http-clients</artifactId>
+                <version>${version.bom.software.amazon.awssdk}</version>
+            </dependency>
+            <dependency>
+                <groupId>software.amazon.awssdk</groupId>
+                <artifactId>metric-publishers</artifactId>
+                <version>${version.bom.software.amazon.awssdk}</version>
+            </dependency>
+            <dependency>
+                <groupId>software.amazon.awssdk</groupId>
+                <artifactId>protocols</artifactId>
+                <version>${version.bom.software.amazon.awssdk}</version>
+            </dependency>
+            <dependency>
+                <groupId>software.amazon.awssdk</groupId>
+                <artifactId>ruleset-testing-core</artifactId>
+                <version>${version.bom.software.amazon.awssdk}</version>
+            </dependency>
+            <dependency>
+                <groupId>software.amazon.awssdk</groupId>
+                <artifactId>service-test-utils</artifactId>
+                <version>${version.bom.software.amazon.awssdk}</version>
+            </dependency>
+            <dependency>
+                <groupId>software.amazon.awssdk</groupId>
+                <artifactId>services-custom</artifactId>
+                <version>${version.bom.software.amazon.awssdk}</version>
+            </dependency>
+            <dependency>
+                <groupId>software.amazon.awssdk</groupId>
+                <artifactId>services</artifactId>
+                <version>${version.bom.software.amazon.awssdk}</version>
+            </dependency>
+            <dependency>
+                <groupId>software.amazon.awssdk</groupId>
+                <artifactId>test-utils</artifactId>
+                <version>${version.bom.software.amazon.awssdk}</version>
+            </dependency>
+            <dependency>
+                <groupId>software.amazon.awssdk</groupId>
+                <artifactId>third-party-slf4j-api</artifactId>
+                <version>${version.bom.software.amazon.awssdk}</version>
+            </dependency>
+            <dependency>
+                <groupId>software.amazon.awssdk</groupId>
+                <artifactId>third-party</artifactId>
+                <version>${version.bom.software.amazon.awssdk}</version>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
```

Comment:
Would it make sense to avoid repetition by either:

1. Importing the normal BOM and overriding just lucene dependencies? But then you'd be managing `hibernate-search-backend-lucene` (not next) even though it can't work with the managed version of Lucene.
2. Importing a separate POM dedicated to just dependencies, not Hibernate Search artifacts, in _both_ platform poms, and overriding the Lucene version here?

Reply:
> Importing a separate POM dedicated to just dependencies, not Hibernate Search artifacts, in both platform poms, and overriding the Lucene version here?

this one would be nice yeah ... but I guess if we'd make some bom/platform-common we'd need to either publish it as well ... or don't publish but inline all the dependencies... 

let me pull the common part and see if it would look better that way, and if so -- I'll just inline the dependencies, after all ... it is to manage dependencies and not to be read by a human on a daily basis 😁 🙈